CHANGELOG.rst

```diff
@@ -31,6 +31,11 @@ Fixed
   (bug fix)
 
   Contributed by carbineneutral. #3534 #3544
+* st2 pack commands now work when StackStorm servers are behind a HTTP/HTTPS proxy. You can set
+  ``http_proxy`` or ``https_proxy`` environment variables for ``st2api`` and ``st2actionrunner``
```

Comment:
Looks like some tools respect lowercase only https://unix.stackexchange.com/a/212972/217844 https://serverfault.com/questions/571887/why-wget-doesnt-read-http-proxy-in-uppercase-format for proxy env vars too.

So not sure, or support lowercase only as others do, or respect both cases.